### DISCUSSION TITLE
事务消息回查相关疑问

### DISCUSSION THREAD

**Discussion by LittleBoy18** [apache/rocketmq#7350] [Q&A]
事务消息回查是每个broker都会去定时执行的吗？如果两个broker同时对producer进行事务消息回查，是不是会造成producer commit两次造成broker上有重复的数据？

**Comment by fuyou001** [comment]
基本不会发生你说的情况。因为sdk发消息时，消息只会发到其中一个broker节点，但在极端场景下，可能同一个消息会发到多个Broker节点上，但在回查事务时，只要应用做好事务回查的幂等逻辑，也不会影响逻辑的正确性。
